src/main/java/slimeknights/tconstruct/common/config/Config.java

```diff
@@ -271,11 +272,30 @@ public static boolean syncConfig() {
       magmaIslandsRate = prop.getInt();
       propOrder.add(prop.getName());
 
+      int[] slimeIslandBlacklist = slimeIslandDimensions;
+
       prop = configFile.get(cat, "slimeIslandBlacklist", slimeIslandBlacklist);
-      prop.setComment("Prevents generation of slime islands in the listed dimensions");
+      prop.setComment("DEPRECATED: Prevents generation of slime islands in the listed dimensions (overrides `slimeIslandDimensions` and `slimeIslandDimensionsIsBlacklist` if modified)");
```

Comment:
Do not add deprecated properties, thats even worse than no migration. `configFile.renameProperty`

Reply:
Apologies, first time touching any Minecrafting mods - not familiar with the API.